<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Target URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_URL">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL Injection Payload</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_Payload">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_CustomPayload">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Test">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Test SQL Injection</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Response:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Response">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>741</width>
      <height>471</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_SaveLogs">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>690</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Logs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ExportLog">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>690</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DarkMode">
    <property name="geometry">
     <rect>
      <x>724</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark Mode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_About">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>710</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>909</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
